src/alt-da/da-server/da_server_launcher.star

```diff
@@ -0,0 +1,96 @@
+shared_utils = import_module(
+    "github.com/ethpandaops/ethereum-package/src/shared_utils/shared_utils.star"
+)
+constants = import_module(
+    "github.com/ethpandaops/ethereum-package/src/package_io/constants.star"
+)
+
+# The dirpath of the data directory on the da-server container
+# note that we use /home which is available but not persistent
+# because we aren't mounting an external kurtosis file
+# this means that the data is lost when the container is deleted
+DATA_DIRPATH_ON_DA_SERVER_CONTAINER = "/home"
+
+# Port IDs
+DA_SERVER_HTTP_PORT_ID = "http"
+
+# Port nums
+DA_SERVER_HTTP_PORT_NUM = 3100
+
+
+def get_used_ports():
+    used_ports = {
+        DA_SERVER_HTTP_PORT_ID: shared_utils.new_port_spec(
+            DA_SERVER_HTTP_PORT_NUM,
+            shared_utils.TCP_PROTOCOL,
+            shared_utils.HTTP_APPLICATION_PROTOCOL,
+        ),
+    }
+    return used_ports
+
+def launch(
+    plan,
+    service_name,
+    da_server_extra_args,
+    generic_commitment,
+):
+
+    config = get_da_server_config(
+        plan,
+        service_name,
+        da_server_extra_args,
+        generic_commitment,
+    )
+
+    da_server_service = plan.add_service(service_name, config)
+
+    http_url = "http://{0}:{1}".format(da_server_service.ip_address, DA_SERVER_HTTP_PORT_NUM)
+    return new_da_server_context(
+        http_url=http_url,
+        generic_commitment=generic_commitment,
+    )
+
+
+def get_da_server_config(
+    plan,
+    service_name,
+    da_server_extra_args,
+    generic_commitment,
+):
+    image = "us-docker.pkg.dev/oplabs-tools-artifacts/images/da-server:devnet"
+    ports = get_used_ports()
+
+    cmd = [
+        "da-server",
+        "--file.path=" + DATA_DIRPATH_ON_DA_SERVER_CONTAINER,
+        "--addr=0.0.0.0",
+        "--port=3100",
+        "--log.level=debug",
+        "--generic-commitment=" + str(generic_commitment),
```

Comment:
`da_server` image complains on this at startup: `Incorrect Usage: flag provided but not defined: -generic-commitment`

I'd image the `latest` image found here `us-docker.pkg.dev/oplabs-tools-artifacts/images/da-server:latest` is stale and missing the flag?

Reply:
Yes confirmed that the latest image is missing the flag. And the more recent one v0.1.0-rc.1 from June gives me this error when I try to run on my mac: `rosetta error: failed to open elf at /lib/ld-musl-x86_64.so.1`

Reply:
WIth the latest kurtosis 1.3 release I was able to add optimism repo as a submodule and build the da-server image locally: [e709d7c](https://github.com/ethpandaops/optimism-package/pull/69/commits/e709d7ce31419aa29ceddd776b7d2858b444bf92)